core/embed/models/T3W1/firmware_flash.jlink

```diff
@@ -0,0 +1,7 @@
+device STM32U5G9ZJ
+if swd
+speed 20000
+loadbin build/firmware/firmware.bin.padded.bin 0x0C044000
```

Comment:
probably would be best to generate these files from model_xx.h, should be similar to what tools/trezor_core_tools/lsgen.py does. Or let's at least create an issue for reworking this.